examples/backends/vllm/launch/agg_multimodal.sh

```
@@ -12,12 +12,14 @@
 # see agg_multimodal_epd.sh
 
 set -e
-trap 'echo Cleaning up...; kill 0' EXIT
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "$SCRIPT_DIR/../../../common/gpu_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/gpu_utils.sh was not specified as input (see shellcheck -x).

(SC1091)